Day3/application.py

```diff
@@ -9,15 +9,23 @@
 Session = sessionmaker(bind = engine)
 session = Session()
  
+@app.route("/admin")
+def admin():
+   usrs =   session.query(User).all()
```

Comment:
does  this show the list in the desc order of the user creation time

Reply:
updated the code sir, will commit after all the changes.